packages/snaps-sdk/src/jsx/validation.test.tsx

```diff
@@ -493,6 +493,26 @@ describe('AddressStruct', () => {
     <Address address="eip155:1:0x1234567890abcdef1234567890abcdef12345678" />,
     <Address address="bip122:000000000019d6689c085ae165831e93:128Lkh3S7CkDTBZ8W7BbpsN3YYizJMp8p6" />,
     <Address address="cosmos:cosmoshub-3:cosmos1t2uflqwqe0fsj0shcfkrvpukewcw40yjj6hdc0" />,
+    <Address
+      address="0x1234567890abcdef1234567890abcdef12345678"
+      truncate={false}
+      avatar={false}
+    />,
+    <Address
+      address="0x1234567890abcdef1234567890abcdef12345678"
+      displayName={true}
+    />,
+    <Address
+      address="0x1234567890abcdef1234567890abcdef12345678"
+      displayName={true}
+      avatar={false}
+    />,
+    <Address
+      address="0x1234567890abcdef1234567890abcdef12345678"
+      truncate={true}
+      displayName={false}
+      avatar={true}
```

Comment:
Can you add a test for invalid props ? Like trying to use this prop with invalid data ?